### ISSUE TITLE
Crash when opening Crafting Grid

### ISSUE THREAD

**Issue by llloyd4** [raoulvdberge/refinedstorage#1335]
#### Issue description:

When I go to open the Crafting Grid to get something out it CTDs. Not sure if it's you or Galacticraft 4, the first 2 lines mention them and the next several mention you and it is only when I go to view the crafting grid that it CTDs. https://github.com/micdoodle8/Galacticraft/issues/2868 - The same issue on the GC 4 github. 

#### What happens:

When I go to open the Crafting Grid to get something out it CTDs.

#### What you expected to happen:

To not CTD when opening the Crafting Grid but show me my items in storage

#### Steps to reproduce:
1. Load up Minecraft
2. Load up my world
3. Open Crafting Grid
...

#### Version (make sure you are on the latest version before reporting):

- Minecraft: 1.10.2
- Forge: 1.10.2-12.18.3.2316
- Refined Storage: 1.2.26

Does this issue occur on a server? [yes/no]

No idea, I am playing single player.

#### If a (crash)log is relevant for this issue, link it here:

[https://pastebin.com/Bc1usRNh]

**Comment by raoulvdberge** [comment]
It's Galacticraft.

**Comment by llloyd4** [comment]
Ok, so, in the above issue posted to here and Galacticraft, you *both* blame the other mod. So. They say your mod is broken, you say their mod is broken. Would someone please figure out the problem and fix it, I really don't care who. :( To quote the above linked issue -

" It's not a bug in our mod! It's something wrong with the item.

Refined Storage is attempting to draw a Tier 1 Rocket item with an item damage value of 7.

That's a damage value which doesn't exist for Galacticraft rockets. There is no way that Galacticraft can make that item, and you won't find it in JEI for example. For rockets, the allowed damage values are 0 through 4 only. "

**Comment by raoulvdberge** [comment]
Yeah RS doesn't do that on its own, I guess your some external mod inserted an invalid stack in the RS system.

**Comment by llloyd4** [comment]
So how do I find it? Since I've crafted nothing for GC4, can't be that. I can't even get into the Crafting Grid with out a CTD so I've been unable to do anything. :( I mean, I might be able to narrow it down to one of the drives as long as what ever it is is contained in one drive and not scattered across two or three. But if viewing that one drive is going to cause a CTD then how do I know what item in that drive is the issue? O.o

**Comment by raoulvdberge** [comment]
Remove drives until you no longer have the crash.

**Comment by llloyd4** [comment]
Right, that's what I am saying, but once I find the drive, how do I find what is in that drive is the problem? O.o

**Comment by raoulvdberge** [comment]
The crash happens on a ItemTier1Rocket. Black-list that with compare NBT and damage off.

**Comment by llloyd4** [comment]
Ok, so, no idea what it was, but I backed up my world, uninstalled GC4, reloaded the world, clicked "Yes" to the "Missing Blocks" , went over everything to find all invalid patterns (I had Advanced Rocketry installed before but decided to switch to GC4, a better documented mod. >.< ) that I might have missed when I removed all the AR items from the game prior to uninstalling AR, shut down, reenabled GC4, and it seems stable so far. Freaking weird. I could have sworn I uninstalled AR and loaded my world before installing GC4 to be sure everything AR related was gone. O.o
